src/Screens/Home.js

```diff
@@ -106,6 +110,7 @@ class Home extends Component {
                   onClick={() => {
                     logout(history.push("/"));
```

Comment:
can you reorder this, redirect should happen at the end, you don't want to redirect and then have function call after, that will cause problems, you are mingling old functionality of old page and new page.